BricksAndHearts.UnitTests/ServiceTests/Admin/AdminServiceTests.cs

```diff
@@ -72,4 +75,169 @@ public void CancelAdminAccessRequest_SetsHasRequestedAdminToFalse_ForCorrectUser
         // Assert
         context.Users.Single(u => u.Id == requestedAdminUser.Id).HasRequestedAdmin.Should().BeFalse();
     }
+
+    [Fact]
+    public void GetAdminLists_GetsListOfCurrentAndPendingAdmins_ForAdminUser()
+    {
+        // Arrange
+        using var context = Fixture.CreateReadContext();
+        var service = new AdminService(context);
+
+        // Act
+        var result = service.GetAdminLists().Result;
+
+        // Assert
+
+        result.Item1.Should().BeOfType<List<UserDbModel>>();
+        result.Item2.Should().BeOfType<List<UserDbModel>>();
+    }
+
+    [Fact]
+    public void GetLandlordDisplayList_CalledWithApproved_ReturnsApprovedLandlords()
+    {
+        // Arrange
+        using var context = Fixture.CreateReadContext();
+        var service = new AdminService(context);
+
+        // Act
+        var result = service.GetLandlordDisplayList("Approved").Result;
+
+        // Assert
+        result.Should().BeOfType<List<LandlordDbModel>>().And.Subject.Where(u => u.CharterApproved).Should()
+            .HaveCount(result.Count);
+    }
+    
+    [Fact]
+    public void GetLandlordDisplayList_CalledWithUnapproved_ReturnsUnapprovedLandlords()
+    {
+        // Arrange
+        using var context = Fixture.CreateReadContext();
+        var service = new AdminService(context);
+
+        // Act
+        var result = service.GetLandlordDisplayList("Unapproved").Result;
+
+        // Assert
+        result.Should().BeOfType<List<LandlordDbModel>>().And.Subject.Where(u => u.CharterApproved == false).Should()
+            .HaveCount(result.Count);
+    }
+    
+    [Fact]
+    public void GetLandlordDisplayList_CalledWithNothing_ReturnsAllLandlords()
+    {
+        // Arrange
+        using var context = Fixture.CreateReadContext();
+        var service = new AdminService(context);
+
+        // Act
+        var result = service.GetLandlordDisplayList("").Result;
+        
+        // Assert
+        result.Should().BeOfType<List<LandlordDbModel>>().And.Subject.Should().HaveCount(context.Landlords.ToList().Count);

```

Comment:
```suggestion
        result.Should().BeOfType<List<LandlordDbModel>>().And.Subject.Should().HaveCount(context.Landlords.Count());
```

nit don't need the ToList. Doesn't really matter in tests but the `ToList` will load all the landlords into memory as a list, whereas `Count()` should do a `SELECT COUNT(*) FROM` which is quicker